pkg/unidling/api/types.go

```diff
@@ -9,6 +9,10 @@ const (
 	// objects associated with the idled endpoints
 	UnidleTargetAnnotation = "idling.alpha.openshift.io/unidle-targets"
 
+	// PreviousScaleAnnotation contains the previous scale of a scalable object
+	// (currently only applied by the idler)
+	PreviousScaleAnnotation = "scale.openshift.io/previous-scale"
```

Comment:
I can, but when I was talking to @smarterclayton, we were discussing the idea of making something like this automatically applied on scales in the future (and then have scale.openshift.io/xyz passed through the scale subresource so that it can act like a prototype API field, just like other objects), so using something like this would future-proof that somewhat.